src/module/Entity/src/Entity/Entity/Entity.php

```diff
@@ -253,6 +253,11 @@ public function isUnrevised()
         return $this->countUnrevised() > 0;
     }
 
+    public function isDead()
+    {
+        return !$this->getHead();
```

Comment:
You could instead instead use [HasCurrentRevisionFilter](https://github.com/serlo-org/athene2/blob/master/src/module/Versioning/src/Versioning/Filter/HasCurrentRevisionCollectionFilter.php)